content/doc/developer/publishing/documentation.adoc

```diff
@@ -38,12 +37,12 @@ Multiple formats are supported: GitHub Flavored Markdown, Asciidoc or raw text.
 To publish the plugin documentation on GitHub:
 
 . Create a README page and put the plugin documentation there.
-  This page will become a landing page for the link:https://plugins.jenkins.io/[Plugin site].
+  This page will become a landing page for the link:https://plugins.jenkins.io/[Plugin site].
```

Comment:
(getting rid of `&nbsp;`)

Reply:
The &nbsp; characters are artifacts of the translation from wiki.jenkins.io.  I find them distracting, so I remove them when I find them.

Reply:
Yes, unfortunately the translator does not do this conversion automatically. I have to do the same manually when converting plugin docs.